src/Controllers/UserController.php

```diff
@@ -22,17 +26,31 @@ public function setService(AbstractService $userService)
 
```

Comment:
We discussed building viewParameters as the function executes for a cleaner code. In this case you render the same view with different parameters. You can write it with one return statement.